src/Input/index.tsx

```diff
@@ -34,6 +36,7 @@ const Input: FunctionComponent<Props> = forwardRef<HTMLInputElement, Props>(func
     name,
     label,
     size = 'lg',
+    rounded = false,
```

Comment:
it looks like we'll have to update all of our inputs to rounded after this change

Reply:
it might be easier to set the default as true

Reply:
Yeah, getting confirmation now it seems like we'll always be using the square inputs for customer facing web apps, with the exception of a few internal apps, after I merge this I will update accordingly